t/discovery/dns/mix.t

```diff
@@ -120,11 +120,3 @@ upstreams:
     }
 --- request
 GET /t
---- grep_error_log eval
-qr/connect to 127.0.0.1:1053/
---- grep_error_log_out
-connect to 127.0.0.1:1053
-connect to 127.0.0.1:1053
-connect to 127.0.0.1:1053
-connect to 127.0.0.1:1053
-connect to 127.0.0.1:1053
```

Comment:
Why do you delete this?

Reply:
Must have been done by mistake while testing something. Pushed the latest commit